### PR TITLE
W-12614796 b2c shop example

### PR DESCRIPTION
**Pull request by sathya0**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by tarcang — DISMISSED** [review]
Thank you! Looks good to me.

File: commerce-cloud-shop-api/2.1/modules/ROOT/pages/shop-api-connector-examples.adoc [hunk 1]
```diff
@@ -76,6 +93,79 @@ image::shop-connector-customer-auth-config.jpg[]
...
+
+[start = 1]
+. Create new Mule application.
+. Click the Configuration XML tab at the base of the canvas.
+. Copy and paste the above code.
+. Save the project.
+. Run the project as a Mule application by right-clicking the project name in Package Explorer and selecting Run As > Mule Application.
+. Navigate to http://localhost:8081/customer/shopper-customers/{version}/organizations/{organizationId}/customers/actions/login, open Postman, and check the response.
+. You should see the JWT token in the response header. Provide this JWT token from Postman as an authorization header parameter to perform operations.
+
```
Comment (tarcang) [inline commerce-cloud-shop-api/2.1/modules/ROOT/pages/shop-api-connector-examples.adoc:168]: These are meant to be two separate xml files (registered_flow.xml and guest_flow.xml). In the page would that show up as two separate back to back xml files?

**Review by tarcang — COMMENTED** [review]
Overall looks good! Just left a question for clarification.

File: commerce-cloud-shop-api/2.1/modules/ROOT/pages/shop-api-connector-examples.adoc (hunk 1 above)
Comment (sathya0) [reply commerce-cloud-shop-api/2.1/modules/ROOT/pages/shop-api-connector-examples.adoc:168]: Yup! It would be two separate XML blocks

**Review by tarcang — APPROVED** [review]

**Review by tasmoore — APPROVED** [review]

**Pull request merged**